examples/00-fluent/mixing_elbow_settings_api.py

```diff
@@ -180,7 +180,7 @@
 # Solve for 150 iterations.
 
 solver.solution.run_calculation.iterate.get_attr("arguments")
-if solver.get_fluent_version() >= "23.1.0":
+if solver.get_fluent_version() >= "23.2.0":
```

Comment:
I think we should not change this one? 

We could otherwise consider just removing this version check and backward compatibility from the example, assuming that examples are always run with the latest released version (e.g. doc builds).